cmd/apis/bundlecrtapis.go

```diff
@@ -48,6 +48,12 @@ var BundleCreateCmd = &cobra.Command{
 		if proxyZip != "" {
 			_, err = apis.CreateProxy(name, proxyZip)
 		} else if proxyFolder != "" {
+			if stat, err := os.Stat(folder); err == nil && stat.IsDir() {
```

Comment:
Be careful. You need to negate the `stat.IsDir()` result for the test to be correct. Same for all other lines.

Reply:
Thanks for spotting that!